api/api.go

```diff
@@ -50,21 +53,32 @@ func (a *Api) Run() {
 
 	routes := mux.NewRouter().StrictSlash(true)
 
-	routes.Handle(a.setupHandler(
-		"/", []rye.Handler{
+	// Necessary so we do not put /version and
+	// /status/check behind access tokens
+	noAuthHandler := rye.NewMWHandler(rye.Config{})
+
+	routes.Handle(
+		"/", handlers.LoggingHandler(os.Stdout, noAuthHandler.Handle([]rye.Handler{
```

Comment:
I see why, but I'm wondering if we could update `setupHandler` to accept different mwHandler's so we can continue using the cleaner .setupHandler() syntax. With the logging handler this looks gross. :man_shrugging: